ios/PurchasesHybridCommon/PurchasesHybridCommon/Purchases+HybridAdditions.swift

```diff
@@ -113,22 +118,6 @@ extension LogLevel {
     )
 }
 
-extension PurchasesAreCompletedBy {
```

Comment:
I moved this to a new `PurchasesAreCompletedBy+HybridAdditions.swift` file to remove this responsibility from `Purchases+HybridAdditions`